keyboards/0118/blacktip/keyboard.json

```
@@ -0,0 +1,222 @@
+{
+    "keyboard_name": "Blacktip",
+    "manufacturer": "iH0118",
+    "maintainer": "iH0118",
+    "url": "https://github.com/iH0118/blacktip",
+    "bootloader_instructions": "Flip the boootloader switch and press the physical RESET button on the back of the PCB, or hold the top-left key when plugging in the board.",
```

Should only document hardware methods and not bootmagic
```suggestion
    "url": "https://github.com/iH0118/blacktip",
    "bootloader_instructions": "Flip the boootloader switch and press the physical RESET button on the back of the PCB.",
```